app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,8 @@ html lang="#{I18n.locale}"
       = content_for?(:title) ? APP_NAME + ' - ' + yield(:title) : APP_NAME
 
     == stylesheet_link_tag 'application', media: 'all'
-    == javascript_include_tag 'application'
+    - unless controller_name == 'home'
+      == javascript_include_tag 'application'
```

Comment:
this feels a little hinky to me, but offhand I'm not sure I know a better way. It does feel like maybe premature optimization, but it's easily correctable if need be, so I'm ok with it.